src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -245,7 +254,12 @@ public void visitLiteralReturn(int maxAssigned) {
          */
         public void checkCount(DetailAST ast) {
             if (checking && maxAllowed != null && count > maxAllowed) {
-                log(ast.getLineNo(), ast.getColumnNo(), MSG_KEY, count, maxAllowed);
+                if (context.isVoidContext) {
```

Comment:
Same here, please use `isVoidContext` instead of `context.isVoidContext`

Reply:
done